src/pages/_app.tsx

```diff
@@ -15,6 +16,11 @@ export default function App({ Component, pageProps }: AppProps) {
     <ChakraProvider theme={theme}>
       <MDXProvider components={MDXComponents}>
         <Layout>
+          <Head>
+            <meta name='viewport' content='width=device-width, initial-scale=1' />
+            <link rel='icon' type='image/x-icon' href='/favicon.ico' />
+          </Head>
+
```

Comment:
@nhsz Do we need this here if we're already placing this within the PageMetadata component which is then used on every page?

Reply:
I think this can be removed. Tested without it, like you said, this is handled in the `PageMetaData` component

Reply:
Makes sense, updated